source/Octopus.Cli/Commands/ApiCommand.cs

```diff
@@ -56,7 +56,7 @@ protected ApiCommand(IOctopusClientFactory clientFactory, IOctopusAsyncRepositor
             this.FileSystem = fileSystem;
 
             var options = Options.For("Common options");
-            options.Add("server=", $"[Optional] The base URL for your Octopus Server - e.g., http://your-octopus/. This URL can also be set in the {ServerUrlEnvVar} environment variable.", v => serverBaseUrl = v);
+            options.Add("server=", $"[Optional] The base URL for your Octopus Server, e.g., 'http://your-octopus/'. This URL can also be set in the {ServerUrlEnvVar} environment variable.", v => serverBaseUrl = v);
```

Comment:
```suggestion
            options.Add("server=", $"[Optional] The base URL for your Octopus Server, e.g., 'https://octopus.example.com/'. This URL can also be set in the {ServerUrlEnvVar} environment variable.", v => serverBaseUrl = v);
```

I always try and subtle push towards https.
Also, the example.com/example.org/example.net domains are perfect for this

Reply:
Strong agree with both.